### DISCUSSION TITLE
SSL working on the domain, but not with "www"

### DISCUSSION THREAD

**Discussion by Etheonor** [vercel/vercel#5673] [General] [answered]
Hello here!

I just linked a domain on Vercel but I have a weird result... 

https://mywebsite.com is fine, but as soon as I add www, it says it's not secure (https://www.mywebsite.com), did I forgot something to add on my main domain provider?

Thank you in advance

**Comment by cobrexus** [comment, marked as answer]
In Vercel domains, add `www.yourdomain.tld` and configure this subdomain, it should work correctly now.

**Reply by Etheonor** [reply]
I think I just discovered that www is a subdomain... thank you for your answer it works now!

**Reply by cobrexus** [reply]
No problem!

It would be a good idea to mark this answer as accepted to close the discussion
